cmake/rocprofiler_config_packaging.cmake

```
@@ -197,9 +197,14 @@ set(CPACK_RPM_PACKAGE_LICENSE "MIT")
 # Prepare final version for the CPACK use
 #
 # -------------------------------------------------------------------------------------- #
-
+# Make proper version for appending
+# Default Value is 99999
+set(ROCM_VERSION_FOR_PACKAGE "99999")
+if(DEFINED ENV{ROCM_LIBPATCH_VERSION})
+    set(ROCM_VERSION_FOR_PACKAGE $ENV{ROCM_LIBPATCH_VERSION})
+endif()
```

```suggestion
set(ROCM_VERSION_FOR_PACKAGE "99999" CACHE STRING "")
if(DEFINED ENV{ROCM_LIBPATCH_VERSION})
    set(ROCM_VERSION_FOR_PACKAGE "$ENV{ROCM_LIBPATCH_VERSION}" CACHE STRING "" FORCE)
endif()
```

Env variables are antithetical to cmake -- they do not support reproducibility and this does not support `-DROCM_VERSION_FOR_PACKAGE=<...>`; this is reproducible and supports defining via the command line.

---

Thanks you @jrmadsen . Updated the same